packages/dm-core-plugins/src/form/types.tsx

```diff
@@ -48,7 +48,7 @@ export type TBooleanFieldProps = {
   uiAttribute?: any
 }
 
-export declare type Variants = 'error' | 'default'
+export declare type Variants = 'error' | 'success' | 'error'
```

Comment:
```suggestion
export declare type Variants = 'error' | 'success' | 'warning'
```
as according to the eds docs https://eds-storybook-react.azurewebsites.net/?path=/docs/inputs-textfield--introduction